database/migrations/2023_01_12_191058_create_invoice_details_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateInvoiceDetailsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('invoice_details', function (Blueprint $table) {
+            $table->bigIncrements('id')->comment('ID');
+            $table->foreignId('invoice_id')->comment("Invoice ID")->constrained()->onDelete('cascade');
+            $table->string('medicines')->comment('Medicines');
```

Comment:
Pls, use Json data type!
in migration,
`$table->string('medicines')->json('field_name')->comment('Medicines');`

in model,
```
   protected $casts = [
        'medicines' => 'array'
    ];
```